media-video/ffmpeg/ffmpeg-4.4.2-r1.ebuild

```diff
@@ -287,6 +287,7 @@ RDEPEND="${RDEPEND}
 "
 
 DEPEND="${RDEPEND}
+	amf? ( media-libs/amf-headers )
```

Comment:
Now this on the other hand doesn't require a revbump to ffmpeg :) rest of this PR works so I guess I'll move and rebase this revbump out. Thanks!